lib/assert.js

```diff
@@ -89,14 +89,25 @@ x.throws = function (fn, err, msg) {
 	}
 
 	try {
+		var errMsg;
 		if (typeof err === 'string') {
-			var errMsg = err;
+			errMsg = err;
 			err = function (err) {
 				return err.message === errMsg;
 			};
 		}
 
 		assert.throws(fn, err, msg);
+
+		if (errMsg) {
+			return Error(errMsg);
```

Comment:
`errMsg` is the expected message. The workaround above stems from #125 and was done because `assert.throws` doesn't take a string argument as the expected message, but we did want that behavior in AVA.

In other words it's not something to be returned.